packages/temporal/activities/mtv_controls_test.go

```diff
@@ -0,0 +1,99 @@
+package activities
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/AdonisEnProvence/MusicRoom/shared"
+	"github.com/stretchr/testify/suite"
+	"go.temporal.io/sdk/testsuite"
+	"go.temporal.io/sdk/worker"
+)
+
+type UnitTestSuite struct {
+	suite.Suite
+	testsuite.WorkflowTestSuite
+
+	env *testsuite.TestActivityEnvironment
+}
+
+func (s *UnitTestSuite) SetupTest() {
+	s.env = s.NewTestActivityEnvironment()
+}
+
+func (s *UnitTestSuite) AfterTest(suiteName, testName string) {
+	s.env = s.NewTestActivityEnvironment()
+}
+
+func (s *UnitTestSuite) Test_Timeout_Tracks_Timer() {
+	ctx := context.Background()
+	// ctx, cancel := context.WithCancel(ctx)
+	totalDuration := 4 * time.Second
+	timer := shared.MtvRoomTimer{
+		State:         shared.MtvRoomTimerStateIdle,
+		Elapsed:       0,
+		TotalDuration: totalDuration,
+	}
+
+	s.env.RegisterActivity(TrackTimerActivity)
+	s.env.SetWorkerOptions(worker.Options{
+		BackgroundActivityContext: ctx,
+	})
+	val, err := s.env.ExecuteActivity(TrackTimerActivity, timer)
+	s.NoError(err)
+
+	var res shared.MtvRoomTimer
+	err = val.Get(&res)
+	s.NoError(err)
+
+	s.T().Logf("%+v\n", res)
```

Comment:
We should clean the test before merging, by removing this line and the comment on line 31.

```suggestion
```